lib/forest_liana/bootstraper.rb

```diff
@@ -152,35 +158,69 @@ def namespace_duplicated_models
         end
     end
 
-    def create_apimap
+    def create_apimap(with_integration=true)
```

Comment:
I don't understand this. Why would we have a different different behaviour according to the environment?

Reply:
My bad